middleware.go

```diff
@@ -0,0 +1,33 @@
+package orlop
+
+import "net/http"
+
+type SecurityHeaderMiddleware struct {
+	handler http.Handler
+}
+
+func NewSecurityHeaderMiddleware(handler http.Handler) http.Handler {
+	return &SecurityHeaderMiddleware{
+		handler: handler,
+	}
+}
+
+func (h *SecurityHeaderMiddleware) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	addSecurityHeaders(w)
+
+	h.handler.ServeHTTP(w, r)
+}
+
+func addSecurityHeaders(w http.ResponseWriter) {
+	addHeaderIfNotExists(w, "Strict-Transport-Security", "max-age=31536000; includeSubDomains; preload")
```

Comment:
A version of this exists in `cors.go`. I'd recommend removing that version there since it was tacked onto unrelated CORS-headers. Either that or generalizing CORS with this somehow.

Reply:
Generalized with existing CORS headers